### PR TITLE
Updating mesosphere-shared-reactjs v0.0.20

### PR DESCRIPTION
**Pull request by mlunoe**
Only change in this version is this: https://github.com/dcos/mesosphere-shared-reactjs/pull/15, which makes sure that the hanging requests actually works.

This also fixes a problem, where the history server would always be set to offline in the beginning.

Before (with network delay):

Before (without network delay):

After:

### CODE REVIEW

File: src/js/events/MesosSummaryActions.js [hunk 1]
```diff
@@ -64,7 +69,7 @@ function requestFromHistoryServer(resolve, reject, timeScale = 'last') {
       requestFromMesos(resolve, reject);
     },
     hangingRequestCallback() {
-      AppDispatcher.handleServerAction({type: REQUEST_MESOS_HISTORY_ONGOING});
+      AppDispatcher.handleServerAction({type: REQUEST_SUMMARY_HISTORY_ONGOING});
```
Comment (mlunoe) [inline src/js/events/MesosSummaryActions.js]: This action type was never defined and therefore never handled (or called before for that matter)

**Comment by mlunoe** [comment]
**mesosphere-ci** Retest this please

**Comment by wavesoft** [comment]
Tested in different network conditions and it LGTM. I can't reproduce the original problem though.

**Comment by mlunoe** [comment]
If you open up `./node_modules/mesosphere-shared-reactjs/utils/RequestUtil.js` and replace the `createCallbackWrapper` with this:
```js
function createCallbackWrapper(callback, requestID) {
  return function () {
    setTimeout(() => {
      setRequestState(requestID, false);

      if (Util.isFunction(callback)) {
        callback.apply(null, arguments);
      }
    }, 2100);
  };
}
```

That will create a delay for some of the requests to trigger the hanging function for one and not for the other.

**Comment by mlunoe** [comment]
Assigning you @rcorral since you have already looked at the code from mesosphere-shared-reactjs.

**Pull request merged**